praktikumsplaner-frontend/frontend/src/components/meldezeitraeume/MeldezeitraumList.vue

```diff
@@ -1,50 +1,107 @@
+                            <v-list-item-title class="justify-center">
+                                {{ meldezeitraum.zeitraumName }}
+                                <slot name="card-title-icon"></slot>
+                            </v-list-item-title>
+                            <v-list-item-subtitle>
+                                <v-icon>mdi-calendar-start</v-icon>
+                                {{
+                                    useFormatter().formatDateFromString(
+                                        meldezeitraum.zeitraum.startZeitpunkt
+                                    )
+                                }}
+
+                                <v-icon>mdi-calendar-end</v-icon>
+                                {{
+                                    useFormatter().formatDateFromString(
+                                        meldezeitraum.zeitraum.endZeitpunkt
+                                    )
+                                }}
+                            </v-list-item-subtitle>
+                        </v-list-item-content>
+                        <v-list-item-action>
+                            <v-btn
+                                icon
+                                @click="warnBeforeDelete(meldezeitraum)"
+                            >
+                                <v-icon>mdi-delete</v-icon>
+                            </v-btn>
+                        </v-list-item-action>
+                    </v-list-item>
+                    <v-divider></v-divider>
+                </v-card>
+            </v-list>
+            <slot
+                v-else
+                name="notfoundmessage"
+            ></slot>
+        </v-col>
+        <yes-no-dialog-without-activator
+            dialogtitle="Achtung!"
+            dialogtext="Wenn Sie den Meldezeitraum löschen, werden auch alle zugehörigen Praktikumsplätze gelöscht. Wollen Sie den Meldezeitraum wirklich löschen?"
+            :value="showDeleteWarningDialog"
+            @no="resetDeleteWarningDialog"
+            @yes="deleteMeldezeitraum"
+        ></yes-no-dialog-without-activator>
+        <progress-circular-overlay
+            :loading="loading"
+        ></progress-circular-overlay>
+    </v-container>
 </template>
 
 <script setup lang="ts">
+import { ref, defineProps, defineEmits } from "vue";
 import Meldezeitraum from "@/types/Meldezeitraum";
 import { useFormatter } from "@/composables/formatter";
+import MeldezeitraumService from "@/api/MeldezeitraumService";
+import YesNoDialogWithoutActivator from "@/components/common/YesNoDialogWithoutActivator.vue";
+import ProgressCircularOverlay from "@/components/common/ProgressCircularOverlay.vue";
+
+const showDeleteWarningDialog = ref(false);
+const loading = ref(false);
+const meldezeitraumToDelete = ref<Meldezeitraum>();
 
 const props = defineProps<{
     value: Meldezeitraum[];
 }>();
+
+const emits = defineEmits<{
+    (e: "delete", meldezeitraum: Meldezeitraum | undefined): void;
```

Comment:
Maybe `deleted` (past form) because the event is thrown when a meldezeitraum was deleted successfully?